src/assets/i18n/de_DE.json

```diff
@@ -153,6 +153,7 @@
   "account.customer.registered.title": "Vielen Dank für Ihre Registrierung!",
   "account.date.month": "Monat",
   "account.date.month.error.required": "Bitte wählen Sie einen Monat aus.",
+  "account.date.title": "Ein Datum wählen",
```

Comment:
Please check if the translation with infinitive form is fine for this title. We did the same for French. Please inform C3PO if any changes are needed.

Reply:
@intershop/c3po-documentation-l10n 'Wählen Sie ein Datum aus' would probably be better at this point